velox/vector/fuzzer/VectorFuzzer.h

```diff
@@ -232,7 +232,8 @@ class VectorFuzzer {
     return boost::random::uniform_01<double>()(rng_) < n;
   }
 
-  // Wraps the given vector in a LazyVector.
+  // Wraps the given vector in a LazyVector. If there are multiple dictionary
+  // layers then the lazy wrap is applied over the innermost dictionary layer.
```

Comment:
> the lazy wrap is applied over the innermost dictionary layer.

My reading of the code suggests that the Lazy is applies to the vector under first layer of dictionary wrapping. Is this not so?

Dict(Dict(Dict(Base)))) -> Dict(Lazy(Dict(Dict(Base)))))

Reply:
there is recursive call at VectorFuzzer.cpp:742 which ensures that the dict gets applied to the innermost layer.
To make this clearer I will modified a test case in vectorFuzzerTest to create and verify this behavior on a 3 layered dictionary.